pmm-app/tests/helper/grafana_helper.js

```diff
@@ -12,7 +12,14 @@ class Grafana extends Helper {
     const { browserContext } = this.helpers.Playwright;
     const basicAuthEncoded = await this.getAuth(username, password);
 
-    await browserContext.setExtraHTTPHeaders({ Authorization: `Basic ${basicAuthEncoded}` });
+    for (let i = 0; i < 20; i++) {
```

Comment:
why are we doing this?

Reply:
I saw sometimes error that `browserContext` is undefined. Then a couple of tests failed on this error.

Reply:
I am not sure, please check again.

Reply:
https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-ui-tests-temp/detail/pmm2-ui-tests-temp/150/tests Here is the failing. This fix solved it.

Reply:
can you please try again, it looks like a temp glitch, probably browserContext was not initialized due to browser crash, please remove and try again.

Reply:
I've deleted it. :)